easybuild/easyconfigs/g/geopandas/geopandas-1.0.1-foss-2023b.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'geopandas'
+version = '1.0.1'
+
+homepage = 'https://geopandas.org'
+description = """GeoPandas is a project to add support for geographic data to pandas objects.
+It currently implements GeoSeries and GeoDataFrame types which are subclasses of pandas.Series
+and pandas.DataFrame respectively. GeoPandas objects can act on shapely geometry objects and
+perform geometric operations."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('SciPy-bundle', '2023.11'),
+    ('Shapely', '2.0.6'),
+    ('Fiona', '1.10.1'),
+    ('pyproj', '3.6.1'),
+    ('matplotlib', '3.8.2'),  # optional
+    ('psycopg2', '2.9.10'),  # optional
+    ('SQLAlchemy', '2.0.29'),  # optional
+    ('networkx', '3.2.1'),  # needed by mapclassify
+    ('scikit-learn', '1.4.0'),  # needed by mapclassify
+    ('numba', '0.58.1'),  # optional, numba extra in mapclassify
```

numba 0.60 for 2023b has been merged, so let's try to use that one
```suggestion
    ('numba', '0.60.0'),  # optional, numba extra in mapclassify
```